doc/Getting-started/index.md

```diff
@@ -65,543 +41,211 @@ To use Kaa Sandbox, your system must meet the following minimum system requireme
-
-Using a set of pre-packaged server-side _log appenders_, the Kaa server is able to store records to a filesystem, a variety of big data platforms (Hadoop, MongoDB, Cassandra, Oracle NoSQL etc.), or submit them directly to a streaming analytics system. It is also possible to [create a custom log appender***]().
-
-The structure of the collected data is flexible and defined by the [log schema***](). Based on the log schema defined for the Kaa application, Kaa generates an object model for the records and the corresponding API calls in the client SDK. Kaa also takes care of data marshalling, managing temporary data storage on the endpoint, and uploading data to the Kaa server.
-
-For programming practice, see [collecting data from endpoints***]().
-
-**Using notifications**
-Kaa uses _notifications_ to distribute structured messages, posted within _notification topics_, from the server to endpoints. A notification structure is defined by a corresponding [notification schema***]().
-
-Endpoint are subscribed to notification topics, which can be either mandatory or optional. Access to notification topics is automatically granted according to the endpoint’s group membership. Notifications can be sent either to every endpoint subscribed to a topic or to an individual endpoint.
-
-Notifications can be assigned expiration timestamps to prevent their delivery after a certain period of time.
-
-For programming practice, see [using notifications***]().
-
-**Distributing operational data**
-Kaa allows you to perform operational data updates, such as configuration data updates, from the Kaa server to endpoints. This feature can be used for centralized configuration management, content distribution, etc. Since Kaa works with structured data and constraint types, it guarantees data integrity.
+Common issues covered in this [guide]({{root_url}}Administration-guide/Troubleshooting/).
 
-The Kaa server monitors the database for changes and distributes updates to endpoints in the incremental form, thus ensuring efficient bandwidth use. The endpoint SDK performs data merging and persistence, as well as notifies the client code about the specific changes made to the data. As a result, the client application knows exactly where in the data structure the changes occurred and can be programmed to react accordingly.
+Please take into account the following known issues and limitations of Kaa Sandbox.
 
-Based on the endpoint’s group membership, it is possible to control what data is available to the endpoint. This is achieved by applying group-specific data overrides, which make it possible to adjust the behavior of the client application based on operational conditions or usage patterns, fine-tune the algorithms according to feedback, implement gradual feature roll-out, A/B testing, etc.
-For programming practice, see [distributing data to endpoints***]().
+* Without the SMTP server configured, you will not be able to create new users. 
```

Comment:
I see dash-prefixed lists are more common in our documentation. It may be worth encoding the rule in our style guide.